tensorflow/tools/ci_build/builds/android_full.sh

```diff
@@ -51,8 +51,11 @@ do
 done
 
 # Build Jar and also demo containing native libs for all architectures.
+# Enable sandboxing so that zip archives don't get incorrectly packaged
+# in assets/ dir (see https://github.com/bazelbuild/bazel/issues/2334)
 echo "========== Building TensorFlow Android Jar and Demo =========="
-bazel build -c opt --fat_apk_cpu=${CPUS} \
+bazel --bazelrc=/dev/null build -c opt --fat_apk_cpu=${CPUS} \
```

Comment:
Could you add a TODO for me to revert these changes once sandboxing is enabled for all builds?